CHANGELOG.txt

```diff
@@ -1,3 +1,7 @@
+Version 0.7.3, 2018-06-01
+-------------------------
+1b01dd0 Update AWS-SDK gem
```

Comment:
Next time, rather than 'Update AWS-SDK gem', I'd more specifically amend this commit to indicate that since aws-sdk v3 modularizes the sdk into service gems, we updated the dependencies to only pull in `aws-sdk-s3` instead of the entire monolith.

Reply:
Gotcha, will do!